components/FlexibleHeader/src/MDCFlexibleHeaderContainerViewController.m

```diff
@@ -96,6 +101,42 @@ - (void)setContentViewController:(UIViewController *)contentViewController {
       [contentViewController didMoveToParentViewController:self];
     }
   }
+
+  [self updateTopLayoutGuideBehavior];
+}
+
+#pragma mark - Enabling top layout guide adjustment behavior
+
+- (void)updateTopLayoutGuideBehavior {
+  if (_topLayoutGuideAdjustmentEnabled) {
+    if ([self isViewLoaded]) {
+      self.contentViewController.view.translatesAutoresizingMaskIntoConstraints = YES;
```

Comment:
Won't this overwrite the contentViewController state?
What if the client has intentionally set translateAutoresizing to NO, or has a different autoresizingMake?

Reply:
Yes, that is intentional. As an owner of this view controller it is our responsibility to lay it out, similar to a view owning a subview.